Cargo.toml

```diff
@@ -1,26 +1,22 @@
 [package]
-name = "alacritty"
+name = "alacritty-core"
```

Comment:
I think calling this `core` is a decent choice, though it's not the most general name (people might not realize that it can be used outside of Alacritty).

Generally I also think that while I think that `-` to separate words looks better, using `_` just makes more sense since it will need to be used as such in the source code anyways.

Reply:
I'm not giving this as much thought right now as it deserves, but I'll just say that in general, the `-core` notion works pretty well when there's high coupling, and generally a "porcelain" and "plumbing" style to the code. Here I think we have something else...

Even `alacritty-backend` might make more sense. Thoughts?

Reply:
lol, fwiw the more I think about "backend" the more I don't like it...

Reply:
I'm starting to think alacritty-core or alacritty-backend is too generic for all the things it would be doing. Maybe we should split it into more crates:

- alacritty-term
- alacritty-render
- alacritty-ptys

Reply:
`alacritty-backend` also seems reasonable, but I think I prefer core since it sounds cooler and it's shorter.

However, I like @tbodt's proposal. I think `alacritty-term` would be a better name than `alacritty-core` because it doesn't sound like it's necessarily Alacritty-internal but it's still short and describes well what it does.

My biggest gripe with that would be that people that don't see the other directories like `alacritty-render` might think that `alacritty-term` is the complete terminal emulator.

Regarding the other names I'm not a huge fan of `alacritty-render` (I can already see me typing renderer and having to correct that) and `alacritty-ptys` should definitely be singular, but since this isn't necessary right now, it's not too important so I'd like to figure out the primary two names first (UI and Term).

I'd also like to mention that using just `alacritty` for frontend is probably a bad idea, since most of the code is likely going to be in the terminal part. So I'm quite happy with the `alacritty-term` name the more I think about it, but the frontend might still need some consideration.

Maybe something like `alacritty-ui`?

Reply:
I'm currently in favor of `alacritty-terminal` and `alacritty-ui`, after much bikeshedding with @chrisduerr in IRC. I also belive personally that `Term` would be better of as `Terminal`, but I don't have super strong feelings.